app/shared/page.tsx

```diff
@@ -0,0 +1,33 @@
+import { Footer } from "@/src/components/Footer";
+import { Header } from "@/src/components/Header";
+import LinkManager from "@/src/components/LinkManager/LinkManager";
+import { END_POINT } from "@/src/constants";
+
+async function getFetchData(url: string) {
+  try {
+    const res = await fetch(url);
+    const result = await res.json();
+    return result;
+  } catch (error) {
+    if (error instanceof Error) {
+      // alert(error.message);
+      console.error(error.message);
+    }
+  }
+}
+
+export default async function page() {
```

Comment:
next에서 컴포넌트 명은 어떻게 표현되어도 사실 상관은 없지만 폴더명이랑 같이 표현해주는게 일반적인 것 같아요~
```suggestion
export default async function SharedPage() {
```